src/test/java/loanbook/model/loan/EmailTest.java

```diff
@@ -21,6 +21,28 @@ public void constructor_invalidEmail_throwsIllegalArgumentException() {
         Assert.assertThrows(IllegalArgumentException.class, () -> new Email(invalidEmail));
     }
 
+    @Test
+    public void isValidGmail() {
+        final String invalidGmail1 = "123456";
+        final String invalidGmail2 = "123456@";
+        final String invalidGmail3 = "123456@gmail";
+        final String invalidGmail4 = "123456@outlook.com";
+        final String invalidGmail5 = "123456@gmail.com@gmail.com";
+        final String validGmail = "example@gmail.com";
```

Comment:
Other valid gmail addresses to try:
`aaaaa.aaaa@gmail.com`
`bbb.aaa.090@gmail.com`

Other invalid gmail addresses to try:
`aaaa.@gmail.com`
`$ymbols@gmail.com`
`symbolB*tween@gmail.com`
`some spaces@gmail.com`

Reply:
Fixed.